### ISSUE TITLE
Discussion: Connect with Rocket.Chat to build a Slack alternative

### ISSUE THREAD

**Issue by GoetheG** [nextcloud/spreed#753]
Due to the fact that Nextcloud is pushing towards a central communication device, wouldn't it be great to combine Nextcloud with Rocket.Chat? Maybe implement Rocket.Chat into Nextcloud and also Nextcloud Talk? Rocket.Chat is open source. Mattermost would be also an alternative. Cause I am missing the chatting function in Nextcloud Talk.

**Comment by GoetheG** [comment]
@nickvergessen What do this tags actually mean? I don’t quite understand. Do you guys consider Rocket.Chat? Thank you! Yours!

**Comment by GoetheG** [comment]
I’d like to add that Rocket.Chat is only an example. Mattermosy is also an option.

**Comment by nickvergessen** [comment]
enhancement means its not a "bug" but a "feature request"
and the other one means its about "integration".

The milestone means it's not being actively worked on or directly developed towards it.

**Comment by ndom91** [comment]
We also run a Rocket.Chat instance - integration in Nextcloud would be great!

**Comment by GoetheG** [comment]
@ndom91, exactly. We are also using both and my college society is sad about the fact that there is no bridge possible. Well, maybe there is a solution. But most users don't have coding lessons and just want to use something they need. Just like the Slack and Dropbox guys. It's easy to setup and easy to use. No coding needed. But open source software is very complicated most of the time. You can't reach those guys who just need simple solutions and have no interest in learning how to code.

**Comment by GoetheG** [comment]
Take a look here: https://rocket.chat/2018/09/17/RC-Nextcloud-partnership-post/

Really looking forward to this partnership. Great work @karlitschek!
